updates/7/updates.sql

```diff
@@ -3,4 +3,52 @@ INSERT INTO ethereum_tokens(address, decimals, protocol) VALUES("0x03Be5C903c727
+*
+UPDATE assets SET cryptocompare="ZORT" where identifier="_ceth_0x825cd4201f8A2bBB1A69668EAc4E5Fa71283273D";
+INSERT INTO ethereum_tokens(address, decimals, protocol) VALUES("0x825cd4201f8A2bBB1A69668EAc4E5Fa71283273D", 18, ""); INSERT INTO assets(identifier, type, name, symbol,started, swapped_for, coingecko, cryptocompare, details_reference) VALUES("_ceth_0x825cd4201f8A2bBB1A69668EAc4E5Fa71283273D", "C", "Zort Coin", "ZORT", 1621883379, NULL, "zort", "ZORT", "0x825cd4201f8A2bBB1A69668EAc4E5Fa71283273D");
+UPDATE assets SET coingecko="shapeshift-fox-token" where identifier="_ceth_0xc770EEfAd204B5180dF6a14Ee197D99d808ee52d";
+INSERT INTO ethereum_tokens(address, decimals, protocol) VALUES("0xc770EEfAd204B5180dF6a14Ee197D99d808ee52d", 18, ""); INSERT INTO assets(identifier, type, name, symbol,started, swapped_for, coingecko, cryptocompare, details_reference) VALUES("_ceth_0xc770EEfAd204B5180dF6a14Ee197D99d808ee52d", "C", "FOX Token", "FOX", 1553639047, NULL, "shapeshift-fox-token", "", "0xc770EEfAd204B5180dF6a14Ee197D99d808ee52d");
+UPDATE assets SET coingecko="overline-emblem" where identifier="_ceth_0xdb0aCC14396D108b3C5574483aCB817855C9dc8d";
+INSERT INTO ethereum_tokens(address, decimals, protocol) VALUES("0xdb0aCC14396D108b3C5574483aCB817855C9dc8d", 8, NULL); INSERT INTO assets(identifier, type, name, symbol,started, swapped_for, coingecko, cryptocompare, details_reference) VALUES("_ceth_0xdb0aCC14396D108b3C5574483aCB817855C9dc8d", "C", "Emblem", "EMB", 1613350627, NULL, "overline-emblem", "", "0xdb0aCC14396D108b3C5574483aCB817855C9dc8d");
+UPDATE assets SET coingecko="ndex" where identifier="_ceth_0x1966d718A565566e8E202792658D7b5Ff4ECe469";
+INSERT INTO ethereum_tokens(address, decimals, protocol) VALUES("0x1966d718A565566e8E202792658D7b5Ff4ECe469", 18, NULL); INSERT INTO assets(identifier, type, name, symbol,started, swapped_for, coingecko, cryptocompare, details_reference) VALUES("_ceth_0x1966d718A565566e8E202792658D7b5Ff4ECe469", "C", "nDEX", "NDX", 1534287453, NULL, "ndex", "", "0x1966d718A565566e8E202792658D7b5Ff4ECe469");
+INSERT INTO ethereum_tokens(address, decimals, protocol) VALUES("0x383518188C0C6d7730D91b2c03a03C837814a899", 9, NULL); INSERT INTO assets(identifier, type, name, symbol,started, swapped_for, coingecko, cryptocompare, details_reference) VALUES("_ceth_0x383518188C0C6d7730D91b2c03a03C837814a899", "C", "Olympus", "OHM", 1616366920, NULL, "olympus", "OHM", "0x383518188C0C6d7730D91b2c03a03C837814a899");
+*
+INSERT INTO ethereum_tokens(address, decimals, protocol) VALUES("0xDa007777D86AC6d989cC9f79A73261b3fC5e0DA0", 9, NULL); INSERT INTO assets(identifier, type, name, symbol,started, swapped_for, coingecko, cryptocompare, details_reference) VALUES("_ceth_0xDa007777D86AC6d989cC9f79A73261b3fC5e0DA0", "C", "DAppNode DAO Token", "NODE", 1626520282, NULL, "dappnode", "", "0xDa007777D86AC6d989cC9f79A73261b3fC5e0DA0");
```

Comment:
The decimals are wrong here.

Should be 18
```suggestion
INSERT INTO ethereum_tokens(address, decimals, protocol) VALUES("0xDa007777D86AC6d989cC9f79A73261b3fC5e0DA0", 18, NULL); INSERT INTO assets(identifier, type, name, symbol,started, swapped_for, coingecko, cryptocompare, details_reference) VALUES("_ceth_0xDa007777D86AC6d989cC9f79A73261b3fC5e0DA0", "C", "DAppNode DAO Token", "NODE", 1626520282, NULL, "dappnode", "", "0xDa007777D86AC6d989cC9f79A73261b3fC5e0DA0");
```